### PR TITLE
add licence, as of issue #16 

### PR DESCRIPTION
**Pull request by Un1Gfn**
add licence, as of #16

### CODE REVIEW

File: LICENCE [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+Redistribution and use in source and binary forms, with or without modification,
+are permitted provided that the following conditions are met:
+
+    * Redistributions of source code must retain the above copyright notice,
+      this list of conditions and the following disclaimer.
+    * Redistributions in binary form must reproduce the above copyright notice,
+      this list of conditions and the following disclaimer in the documentation
+      and/or other materials provided with the distribution.
+    * Neither the name of {{ project }} nor the names of its contributors
```
Comment (dham) [inline LICENCE]: I guess we need `sphinx-contrib/youtube` in here instead of `{{project}}`

**Review by dham — CHANGES_REQUESTED** [review]
Thanks for this. As you can see, I had just one tiny comment.

**Comment by Un1Gfn** [comment]
oopsie :P

File: LICENCE (hunk 1 above)
Comment (Un1Gfn) [reply LICENCE]: already fixed

forgot to comment

**Comment by Un1Gfn** [comment]
fixed  
(see `fix missed template b2e0ee7` above)

**Pull request merged**